applications/FluidDynamicsApplication/python_scripts/apply_two_fluids_inlet_process.py

```diff
@@ -0,0 +1,219 @@
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication as KratosFluid
+
+def Factory(settings, Model):
+    if( not isinstance(settings, KratosMultiphysics.Parameters) ):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ApplyTwoFluidsInletProcess(Model, settings["Parameters"])
+
+
+class ApplyTwoFluidsInletProcess(KratosMultiphysics.Process):
+    def __init__(self, Model, settings):
+        KratosMultiphysics.Process.__init__(self)
+
+        # settings for inlet with interface between fluids and separate velocities
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "mesh_id" : 0,
```

Comment:
This is in the other processes because of legacy reasons. You can remove it in this new one.

Reply:
Removed.